index.html

```diff
@@ -24,14 +24,205 @@ <h1 class="headline1">Hey There.<br>I'm Femi</h1>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="workgrid4">
+                    <p class="workgridp1">Profesional Art Printing Data</p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="workgrid4">
+                    <p class="workgridp1">Profesional Art Printing Data</p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="workgrid4">
+                    <p class="workgridp1">Profesional Art Printing Data/p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                <div  class="workgrid4">
+                    <p class="workgridp1">Profesional Art Printing Data</p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme">
+            <div class="aboutmain">
+            <div class="aboutmain1">
+                <h2>About me</h2>
+                <p class="aboutp">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button">Get My Resume</button>
```

Comment:
- [ ] The Get My Resume button should follow the button styling: Active, Hover and Pressed (see design)